test/test_files/tinysnb/copy/copy_from.test

```diff
@@ -0,0 +1,88 @@
+-GROUP TinySnbReadTest
+-DATASET CSV tinysnb
+
+--
+
+-CASE CopyFromSubquery
+
+-STATEMENT CREATE NODE TABLE A(name STRING, PRIMARY KEY (name));
```

Comment:
Can you add a test where we copy from a pandas data frame and copy from a npy file?